src/importexport/musicxml/internal/musicxml/exportmusicxml.cpp

```diff
@@ -21,11 +21,11 @@
  */
 
 /**
- MusicXML export.
+ MusicXml export.
  */
 
 // TODO: trill lines need to be handled the same way as slurs
-// in MuseScore they are measure level elements, while in MusicXML
+// in MuseScore they are measure level elements, while in MusicXml
```

Comment:
Hm, maybe this change should not apply to comments / natural language. In natural languages, we should of course just use the official spelling, i.e. JSON, MIDI, MusicXML, but only in CamelCase that becomes `Json`, `Midi`, `MusicXml`,